specs/6002-MDET-market-details.md

```diff
@@ -77,35 +79,35 @@ When I look into market info I **Must** see following specification:
   - Lifetime limit
   - Infrastracture fee account balance
   - Global reward pool account balance
-- Metadata: (<a name="6002-MDET-207" href="#6002-MDET-207">6002-MDET-207</a>)
+- Metadata: (<a name="6002-MDET-208" href="#6002-MDET-208">6002-MDET-208</a>)
```

Comment:
@bwallacee @daro-maj @radoslaw-szpiech 

Would you prefer these codes left alone and new ones incremented?

Reply:
alright, updated!